### ISSUE TITLE
Random Delay Before the start of each map to prevent Dual-Screening

### ISSUE THREAD

**Issue by dustdustinthewind** [PulsarcGame/Pulsarc#26]
Due to the nature of Pulsarc and Intralism, it is much harder to read than traditional Vertical Scrolling Rhythm Games (VSRGs). However, Pulsarc/Intralism share similar mechanics with other VSRGs, such as note placement on 4 separate tracks. If someone converts a Pulsarc map to a VSRG, and then play the VSRG on one window/screen while having Pulsarc the active window/screen allows a player to "Dual-Screen." This is when a player uses a VSRG to read a map, while their inputs going into Pulsarc. This is not a practice we want to encourage, so adding a random delay within, say, a .5-2 second (Correction: 100ms and 400ms) second window, can make it harder for Dual-Screening to be effective.

https://discordapp.com/channels/600885985219444741/600886496677068825/601222278448939009
Edit, image reference
![image](https://user-images.githubusercontent.com/16842122/61769943-d4ea2c80-ada0-11e9-811d-bd7e747d31b3.png)

**Comment by jaanjah** [comment]
This solution would not prevent dual-screening and in my opinion is not the way how we should handle dual-screening, I don't have a better suggestion at the moment, but this is definitely not the solution to this.

**Comment by dustdustinthewind** [comment]
".5-2 seconds" was incorrect, Adri originally said 100ms to 400ms, which seems much more reasonable. I have updated with an image of the original image too. However, I assume you have other reasons for not liking this method?

**Comment by jaanjah** [comment]
I think I misunderstood the suggestion. If the delay is randomized everytime you launch the map, then it would probably work. But I think dual-screening would still be a problem even with this fix, just harder to do.

**Comment by dustdustinthewind** [comment]
Yeah, randomized every time, so it doesn't become predictable. And yeah, the idea is to make dual-screening inconsistent and frustrate those users without interfering with normal players too much.

**Comment by Adrriii** [comment]
Dual screening in this scenario would need several map restart. It would still be possible for solo play, but useless in multiplayer (and then tournaments). That's already a great progress.

I have ideas to prevent this in solo as well using only this feature, but maybe that wouldn't be discussed publicly or it would be useless :)
